.env

```diff
@@ -4,5 +4,6 @@ export SECRET_KEY_BASE=aBi07mDCHwwdqkunV/mx7LlDilXqy7GeZlrXMOm1Qxtdx9e6KfZ2BsF7O
 export TARGET_EMAIL=jackacarlisle@gmail.com
 export SES_SERVER=email-smtp.eu-west-1.amazonaws.com
 export SES_PORT=25
-export SMTP_USERNAME=AKIAIUWYA3RHAHIAP62Q
-export SMTP_PASSWORD=Apv5OSMNgaqncYUlfl2VLnIsklyULMBI9i50VNEhEWA7
+export SMTP_USERNAME=AKIAI7P76NEEEBT6QGKQ
```

Comment:
@markwilliamfirth a contract is _totally_ meaningless if _customer_ personally identifiable data is breeched/leaked.
https://github.com/dwyl/hq/issues/221#issuecomment-303990845

@jackcarlisle please remedy. thanks. 👍